robotbase/src/main/java/frc/robot/controls/DriverControls.java

```diff
@@ -0,0 +1,127 @@
+package frc.robot.controls;
+
+import edu.wpi.first.wpilibj.GenericHID.RumbleType;
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj.XboxController.Axis;
+import edu.wpi.first.wpilibj.XboxController.Button;
+import edu.wpi.first.wpilibj2.command.button.JoystickButton;
+import edu.wpi.first.wpilibj2.command.button.POVButton;
+import frc.robot.Robot;
+import frc.robot.controls.util.AxisThresholdTrigger;
+import frc.robot.controls.util.RumbleInterface;
+
+public class DriverControls implements RumbleInterface {
+
+  private XboxController m_controller;
```

Comment:
Should be using a `CommandXboxController` so that you do not have to define each `JoystickButton`. The class creates them for you.